spec/controllers/idv/image_uploads_controller_spec.rb

```diff
@@ -508,7 +510,7 @@
               :idv_document_upload_submitted,
               success: false,
               document_state: 'ND',
-              document_number: nil,
```

Comment:
Non-blocking nit: I know this inconsistency existed before this PR, but I don't love that the specs in this file use both hardcoded values and variables.

Reply:
Agreed.